From 9789fee4dc66b1b945c641fdc044a4c4b406bbd9 Mon Sep 17 00:00:00 2001
From: David Andrey <david.andrey@netmodule.com>
Date: Fri, 4 Jul 2014 11:15:11 +0200
Subject: [PATCH 7/7] net: phy: disable buggy pause feature of micrel KSZ9031

Pause and asychn pause of Micrel KSZ9031 is buggy, so
do not mark the feature as supported

Signed-off-by: David Andrey <david.andrey@netmodule.com>
---
 drivers/net/phy/micrel.c |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/net/phy/micrel.c b/drivers/net/phy/micrel.c
index 412d144..3e2f063 100644
--- a/drivers/net/phy/micrel.c
+++ b/drivers/net/phy/micrel.c
@@ -466,8 +466,9 @@ static struct phy_driver ksphy_driver[] = {
	.phy_id		= PHY_ID_KSZ9031,
	.phy_id_mask	= 0x00fffff0,
	.name		= "Micrel KSZ9031 Gigabit PHY",
-	.features	= (PHY_GBIT_FEATURES | SUPPORTED_Pause
-				| SUPPORTED_Asym_Pause),
+	/* @@NetModule AG, da, buggy pause feature, no link can be setup up with it */
+	.features	= (PHY_GBIT_FEATURES /*| SUPPORTED_Pause
+				| SUPPORTED_Asym_Pause*/),
	.flags		= PHY_HAS_MAGICANEG | PHY_HAS_INTERRUPT,
	.config_init	= kszphy_config_init,
	.config_aneg	= genphy_config_aneg,
--
1.7.10.4
